Source/DafnyCore/AST/TopLevelDeclarations.cs

```diff
@@ -33,13 +33,15 @@ public abstract class Declaration : RangeNode, IAttributeBearingDeclaration, IDe
   private bool scopeIsInherited = false;
 
   public override IEnumerable<AssumptionDescription> Assumptions() {
-    if (Attributes.Contains(Attributes, "axiom")) {
+    if (Attributes.Contains(Attributes, "axiom") &&
```

Comment:
Magic strings like `axiom` and `auto_generated` should be avoided in favor of referring to a single constant where the string is defined.

Reply:
Ditto my previous comment.